built-in-services/account/src/types.rs

```diff
@@ -0,0 +1,172 @@
+use bytes::Bytes;
+use serde::{Deserialize, Serialize};
+
+use protocol::fixed_codec::{FixedCodec, FixedCodecError};
+use protocol::traits::Witness;
+use protocol::types::{Address, Hex, TypesError};
+use protocol::ProtocolResult;
+
+pub const ACCOUNT_TYPE_PUBLIC_KEY: u8 = 0;
```

Comment:
If using a SignatureType enum, readability and maintainability will be better.

Reply:
We thought about that before，Jiayu thinks it is better to use 'u8' to expandability. @yejiayu